benefits/eligibility/templates/eligibility/index.html

```diff
@@ -20,8 +20,14 @@ <h1>{% translate "Choose the transit benefit you would like to enroll in" %}</h1
   </div>
 {% endblock headline %}
 
-{% block explanatory-text %}
-{% endblock explanatory-text %}
+{% block explanatory-text-wrapper %}
+  <div class="row justify-content-center">
+    <div class="col-lg-8">
```

Comment:
How this code works:

In _mobile_, the `col-lg-8` defaults to a full 12-column row. There's no centering necessary.

In _desktop_, the `col-lg-8` creates a 8-column wide div, and the `row justify-content-center` centers the entire div. A `col-lg-8` without the `row justify-content-center` parent will be left-aligned.